cogs/shop.py

```diff
@@ -71,17 +71,17 @@ def shopEmbedCheck(r, u):
                 return sameMessage and ((str(r.emoji) == '✅') or (str(r.emoji) == '❌')) and u == author
 
             #If player is trying to buy spell scroll, search for spell scroll in DB, and find level it can be bought at 
-            if "spell scroll" in buyItem.lower():
+            if "spell scroll" or "potion" in buyItem.lower():
```

Comment:
this is going to make all potions "not exist". You'll have to do this in a separate line or something, because it's going through the "spells" db to look up the spell